src/server/src/test/java/io/cassandrareaper/management/http/HttpCassandraManagementProxyTest.java

```diff
@@ -83,4 +109,98 @@ private <T extends Object> T jsonFromResourceFile(String filename, Class<T> claz
     return new ObjectMapper().readValue(
         this.getClass().getResource(filename).openStream(), clazz);
   }
+
+  @Test
+  public void testRepairHandlers() throws Exception {
+    HttpManagementConnectionFactory connectionFactory = Mockito.mock(HttpManagementConnectionFactory.class);
+    DefaultApi mockClient = Mockito.mock(DefaultApi.class);
+    doReturn("repair-123456789").when(mockClient).repair1(any());
+    ScheduledExecutorService executorService = Mockito.mock(ScheduledExecutorService.class);
+    doReturn(ConcurrentUtils.constantFuture(null)).when(executorService).submit(any(Callable.class));
+
+    HttpCassandraManagementProxy httpCassandraManagementProxy = new HttpCassandraManagementProxy(
+        Mockito.mock(MetricRegistry.class), "",
+        Mockito.mock(InetSocketAddress.class), executorService, mockClient);
+    when(connectionFactory.connectAny(any())).thenReturn(httpCassandraManagementProxy);
+
+    RepairStatusHandler repairStatusHandler = Mockito.mock(RepairStatusHandler.class);
+
+    int repairNo = httpCassandraManagementProxy.triggerRepair(BigInteger.ZERO, BigInteger.ONE, "ks",
```

Comment:
Isn't that part of your PR? In this test those values mean nothing, they're simply ignored.

Reply:
I think it only applies to Cassandra < 2.2 from memory, that's why I'm wondering if it is a good idea to merge this now, since additional work might yield a slightly different solution for parts of this PR.

Reply:
I mean this PR doesn't care even if it's triggerRepair() and no parameters at all. That has no impact to functionality here.